c-boyle/future-hero
# UI Overhaul

## Changes:
- Add updated main menu.
- Add updated control menu. When you open the controls menu, the control type you're using will be displayed first automatically.
- Add settings menu with `Master Volume` and `Camera Sensitivity` sliders, and a toggle that enables an in game timer display to keep track of how long you've been in a level.
- Add updated pause menu.
- Add a new font for the controls prompts.

## Comments:
- The pause menu is missing a "Restart" button because I initially forgot to ask for one... It should arrive sometime Sunday and then I'll add it very quickly.

## Screenshots:
Pause Menu:
![image](https://user-images.githubusercontent.com/62191940/205479556-017ece58-6dc0-429f-bba2-4c60176d5ba5.png)

Controls Menu (Keyboard):
![image](https://user-images.githubusercontent.com/62191940/205479576-6abaaa2d-8dcb-4e88-9f1e-7961d0855c2a.png)

Controls Menu (Gamepad):
![image](https://user-images.githubusercontent.com/62191940/205479584-622def87-7718-463b-a4a2-1af201cf8392.png)

Settings Menu:
![image](https://user-images.githubusercontent.com/62191940/205479596-ab83a383-f890-4737-b672-992b8302f4ea.png)

Main Menu:
![image](https://user-images.githubusercontent.com/62191940/205479602-bbfbbdce-c24b-4906-b7f7-67af111ad756.png)

New Prompt Font:
![image](https://user-images.githubusercontent.com/62191940/205479644-8a68e0a6-9d16-411d-a3f4-5f428a209a24.png)

Video Of Main Menu Navigation:

https://user-images.githubusercontent.com/62191940/205479734-1dcbbc38-c29d-4d18-b092-d78014501221.mp4








### How to Test:
Try out all the new menus, make sure every button does what it should, and that it works well with a controller. 

### Impacted Features:
All UI.
